install/kubernetes/helm/istio/values.yaml

```diff
@@ -78,6 +78,10 @@ tracing:
 #
 kiali:
   enabled: false
+  # username & password for kiali
+  # if no password is set a 40 digit random password will be set
+  username: admin
+  passphrase: ""
```

Comment:
please place this in the charts/kiali/values.yaml.  Only globals, and enabled/disabled defaults should be listed in values.yaml.  This isn't documented anywhere in our repo - so no harm 👍

Reply:
@monotek ya - i'd like to get rid of our custom implementations of Grafana and Prometheus, but I'm not sure if it is possible.  We have direct integration in our own charts to work well with Istio.  Before we can do any of that, we need `helm dep udpate` working - see PR: https://github.com/istio/istio/pull/9306